docs/source/guide/rem-1-intro.md

```diff
@@ -86,8 +86,31 @@ error = abs((ideal_value - noisy_value)/ideal_value)
 print(f"Error without mitigation: {error:.3}")
 ```
 
+## Apply Postselection
+The simplest version of readout error mitigation that we could do is to postselect specific bitstrings using 
+{mod}`mitiq.rem.post_select`. In our case we know that we should only have bitstrings of `[1,1]`, so we could discard
+any results that don't match.
```

Comment:
Good idea adding a subsection on post-selection!

However, I am not sure about the post-selection example. Post-selecting on the expected ideal result is a too strong type of selection :smile: . Basically in this case the q. computer is completely useless. 

What about something a bit more realistic as this (feel free to rewrite in your own words): 

_We observe that the circuit in our example is symmetric with respect to interchanging the two qubits. Assuming we are given the promise that the ideal result must be a unique bitstring, such a bitstring must be symmetric with respect to a bit swap. Therefore, a possible error mitigation strategy is to discard all results with different bits._

If you agree with this, the associated code cell must be changed accordingly.

Reply:
Thanks, I incorporated these changes, but made a slight tweak to put the language in the affirmative since that is how you specify the selector w/ post_select.